src/tests/system/tests/test_ldap.py

```diff
@@ -195,7 +197,7 @@ def test_ldap__user_with_whitespace(client: Client, ldap: LDAP):
 @pytest.mark.ticket(bz=1507035)
 @pytest.mark.topology(KnownTopology.LDAP)
 @pytest.mark.parametrize("method", ["su", "ssh"])
-def test_ldap__password_change(client: Client, ldap: LDAP, method: str):
+def test_ldap__change_password_when_ldap_pwd_policy_is_set_shadow(client: Client, ldap: LDAP, method: str):
```

Comment:
There seem to be some test_ldap.py tests before this one that had name changes in the other PR.   Can those be pulled in with this too?